.circleci/config.yml

```diff
@@ -103,6 +103,48 @@ jobs:
       - setup_remote_docker
       - run: *examples
 
+  golang-1.14:
+    docker:
+      - image: circleci/golang:1.14
+    environment:
+      GOLANG_VERSION: "1.14"
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-1.13{{ checksum "go.sum" }}
```

Comment:
Do we want to reuse the cache from 1.13?

Reply:
Sorry for the sloppy copy heh!